config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -55,6 +55,9 @@ en:
             multiplier: 'Multiplier'
             question: 'Question'
             grade_summary: 'Grade Summary'
+            publish_grade_hint: >
+              The grades are not shown to the student. You can publish the grades at the submissions
```

Comment:
Oh sorry, misread this - then `The grades are currently not viewable to the student. You can publish ...`